src/__tests__/lib/containers/drug_upload_tool/UserFileGrid.test.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react'
+import { shallow } from 'enzyme'
+import formschemaJson from '../../../../mocks/formschema.json'
+import UserFileGrid, { UserFileGridProps } from '../../../../lib/containers/drug_upload_tool/UserFileGrid'
+import { mockUserProfileData } from '../../../../mocks/mock_user_profile'
+import { mockFileEntity } from '../../../../mocks/mock_file_entity'
+
+
+const SynapseClient = require('../../../../lib/utils/SynapseClient')
+const targetFolderId = 'syn9988882982'
+const token: string = '123444'
+const pathpart = 'someTool'
+const parentContainerId: string = 'syn20355732'
+
+
+const createShallowComponent = async (props: UserFileGridProps, disableLifecycleMethods: boolean = false) => {
+  const wrapper = await shallow(
```

Comment:
can do  `const wrapper = await shallow<UserFileGrid>(...) ` so you can remove the cast below on line 27